File: pqcserver/src/main/java/com/auth/pqcserver/dto/RegistrationResponseDto.java
package com.auth.pqcserver.dto;

import lombok.Data;

@Data
public class RegistrationResponseDto {
    private String signatureDilithium;
    private String signatureRSA;

    private String publicKeyDilithium;
    private String publicKeyRSA;

    private Long userid;

    public RegistrationResponseDto() {}

}

File: pqcclient/app/src/main/java/com/fidoauth/pqcclient/dto/RegistrationResponseDto.kt
package com.fidoauth.pqcclient.dto

data class RegistrationResponseDto(
    val userid: Long,
    val challenge: String,
    val publicKeyRSA: String,
    val signatureRSA: String,
    val publicKeyDilithium: String,
    val signatureDilithium: String
)
